platform-dom/src/main/resources/com/softicar/platform/dom/elements/dom-button-style.css

```diff
@@ -21,7 +21,6 @@
 	flex-direction: row;
 	padding: 0;
 	position: relative;
-	white-space: nowrap;
```

Comment:
I don't think we should do this. Buttons with multi-line labels are _not_ a thing, in any UI I ever came across.

Instead, I think `white-space: nowrap;` should be retained, and `text-overflow: ellipsis;` should be added.

Reply:
Doesnt work unfortunately